modules/vpc/policy-developer.tf

```diff
@@ -18,6 +18,7 @@ data "aws_iam_policy_document" "developer" {
       "ec2:DescribeSecurityGroups",
       "ec2:DescribeVpcs",
       "ec2:DescribeSubnets",
+      "ec2:DescribeNetworkInterfaces",
```

Comment:
Please add this specific permission in https://github.com/FormidableLabs/terraform-aws-serverless/blob/1bd1a49ecf43a3b8f48103a4b04c5cea2801e493/README.md#iam-notes under the discussion of privileges that are still wildcarded -- we enumerate them all so a consumer knows what is _not_ locked down when using this module ;)